plugin/src/steps/source-nodes/update-nodes/content-update-interval.js

```diff
@@ -11,7 +11,8 @@ import { LAST_COMPLETED_SOURCE_TIME } from "../../../constants"
  */
 const checkForNodeUpdates = async ({ cache, emitter }) => {
   // get the last sourced time
-  const since = await cache.get(LAST_COMPLETED_SOURCE_TIME)
+  const lastCompletedSourceTime = await cache.get(LAST_COMPLETED_SOURCE_TIME)
+  const since = lastCompletedSourceTime - 500
```

Comment:
Where does the 500 value come from here?

Reply:
It's somewhat arbitrary - often we were missing node updates by a few milliseconds during development. It has no effect on production builds but makes development content updates reliable. We actually had this in here for a really long time and I removed it before the holidays when Jason did his big refactor. I thought the refactor had removed the need for this but it's still an issue.

Reply:
If we switched from using a timestamp to a cursor this wouldn't be an issue anymore, but that's too big of a change right now :)

Reply:
This is the problem -> https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/issues/386